index.md

```diff
@@ -31,7 +31,7 @@ We've currently established links with a number of organizations:
 * **Develop & maintain a common base of IT Tools** for the main branches of the Social Economy (currently mainly Odoo modules).
 * **Maintain documentation** for the tools we build, making it as easy as possible for others to install and use our tools.
 * **Promote Open Source tools** and support their community.
-* **Improve Cooperativist movement through IT**. We truly believe that the Cooperativism movement can improve our society and take care of our environment.
+* **Improve Cooperativist movement through IT**. We truly believe that the Cooperativism movement can improve our society and our environment.  # a bit vague
```

Comment:
I admit. You can definitely make this less cheeky if you want. Removing the 'truly' can be a start. :)

Reply:
I'd forgotten to submit my comments which were a bit more verbose and polite than "a bit vague", sorry :-)